### PR TITLE
chore: More SFTP/FTP/SMB Detailed Instructions

### PR DESCRIPTION
**Pull request by porisius**
More detailed information for SFTP/FTP/SMB Mod Installations

### CODE REVIEW

**Review by mircearoata — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 1]
```diff
@@ -87,6 +91,56 @@ Note that applying changes in ficsit-cli is a global action -
...
+=== FTP - File Transfer Protocol
+
+File transfer protocol (FTP) is a common, but outdated method to transfer files over the Internet/Network. 
+The commonly used port number is TCP/21, however, refer to your provider for any specific configuration setups. 
+The examples provided are for a self-hosted dedicated server.
+
+The username used requires Read/Write/Delete or Read/Modify permissions. Refer to your provider for any issues 
+accessing the Dedicated Server location.
+
+The path required for FTP to be used is the following: Refer to the examples for more information.
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 2]
```diff
@@ -87,6 +91,56 @@ Note that applying changes in ficsit-cli is a global action -
...
+The commonly used port number is TCP/445, however, refer to your provider for any specific configuration setups. 
+The examples provided are for a self-hosted dedicated server.
+
+The username used requires Read/Write/Delete or Read/Modify permissions. Refer to your provider for any issues accessing the Dedicated Server location.
+
+The path required for FTP to be used is the following: Refer to the examples for more information.
+`\\ServerNameOrIP\ShareName\Path`
+
+Ficsit-CLI Example
+image:DedicatedServers/CLI_SMB.png[Ficsit-CLI Example]
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/images/DedicatedServers/SMM_FTP.png
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/images/DedicatedServers/SMM_FTP.png:1]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 3]
```diff
@@ -61,7 +61,9 @@ If using the Mod Manage to manage your server:
 1. xref:ForUsers/SatisfactoryModManager.adoc[Download and install the Satisfactory Mod Manager].
 2. Open the Mod Manager.
 3. In the left panel, under Other, select "Manage Servers".
-4. Enter server connection details in the popup window, for example, select `ftp://` and enter `user:password@host/path`.
+4. If the server is not hosted locally to ficsit-cli, enter the server connection details. 
+  Refer to link:#FileTransferProtocols[File Transfer Protocols section] for more details. 
+  Otherwise, use the file system path necessary. Ex D:\SatisfactoryDS or /opt/SatisfactoryDedicatedServer
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 4]
```diff
@@ -76,7 +78,9 @@ If using ficsit-cli to manage your server:
    depending on which environment you are more comfortable working with.
 2. Run the application in a terminal window.
 3. Navigate to the "Installations" > "New Installation" screen.
-4. Enter the server connection details, for example `//192.168.1.42/appdata/satisfactory` or `ftp://user:password@host/path`.
+4. If the server is not hosted locally to ficsit-cli, enter the server connection details. 
+  Refer to link:#FileTransferProtocols[File Transfer Protocols section] for more details. 
+  Otherwise, use the file system path necessary. Ex D:\SatisfactoryDS or /opt/SatisfactoryDedicatedServer
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 5]
```diff
@@ -87,6 +91,56 @@ Note that applying changes in ficsit-cli is a global action -
...
+The path required for FTP to be used is the following: Refer to the examples for more information.
+`sftp://username:password@ServerNameOrIP:Port/path/`
+
+Satisfactory Mod Manager Example
+image:DedicatedServers/SMM_SFTP.png[Satisfactory Mod Manager Example]
+
+Ficsit-CLI Example
+image:DedicatedServers/CLI_SFTP.png[Ficsit-CLI Example]
+
+=== SMB/CIFS - Server Message Block/Common Internet File System _(i.e. Windows Network Share)_ *CURRENTLY FICSIT-CLI ONLY*
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 6]
```diff
@@ -87,6 +91,56 @@ Note that applying changes in ficsit-cli is a global action -
...
+
+Server Message Block (SMB) is also commonly known as Windows File Shares or CIFS (Common Internet File System) is a common method to transfer files over a Network, commonly on Windows Systems, but may be used on Linux/Unix Systems. 
+The commonly used port number is TCP/445, however, refer to your provider for any specific configuration setups. 
+The examples provided are for a self-hosted dedicated server.
+
+The username used requires Read/Write/Delete or Read/Modify permissions. Refer to your provider for any issues accessing the Dedicated Server location.
+
+The path required for FTP to be used is the following: Refer to the examples for more information.
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 7]
```diff
@@ -87,6 +91,56 @@ Note that applying changes in ficsit-cli is a global action -
...
+
+=== SFTP - Secure File Transfer Protocol
+
+Secure File transfer protocol (SFTP) is a common method to transfer files over the Internet/Network. 
+The commonly used port number is TCP/22, however, refer to your provider for any specific configuration setups. 
+The examples provided are for a self-hosted dedicated server.
+
+The username used requires Read/Write/Delete or Read/Modify permissions. Refer to your provider for any issues accessing the Dedicated Server location.
+
+The path required for FTP to be used is the following: Refer to the examples for more information.
```
[the remark this answers is not in the document]
Comment (porisius) [reply modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc]: https://github.com/satisfactorymodding/Documentation/pull/272/commits/aa15f4300d61c981ce9e6673dd1bfbd609fe5dab

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 8]
```diff
@@ -78,9 +81,10 @@ If using ficsit-cli to manage your server:
    depending on which environment you are more comfortable working with.
 2. Run the application in a terminal window.
 3. Navigate to the "Installations" > "New Installation" screen.
-4. If the server is not hosted locally to ficsit-cli, enter the server connection details. 
-  Refer to link:#FileTransferProtocols[File Transfer Protocols section] for more details. 
-  Otherwise, use the file system path necessary. Ex D:\SatisfactoryDS or /opt/SatisfactoryDedicatedServer
+4. Enter server connection details in the popup window.
+  a. If the server is not hosted locally to ficsit-cli, enter the server connection details. 
+   - link:#FileTransferMethods[File Transfer Methods] section has more information regarding FTP/SFTP/SMB configurations.
```
Comment (mircearoata) [inline modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc:86]:
```suggestion
   - The link:#FileTransferMethods[File Transfer Methods] section has more information regarding FTP/SFTP/SMB configurations.
```

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 9]
```diff
@@ -56,15 +56,18 @@ You could also install ficsit-cli on the server and interact with its file syste
...
 
 1. xref:ForUsers/SatisfactoryModManager.adoc[Download and install the Satisfactory Mod Manager].
 2. Open the Mod Manager.
 3. In the left panel, under Other, select "Manage Servers".
-4. If the server is not hosted locally to ficsit-cli, enter the server connection details. 
-  Refer to link:#FileTransferProtocols[File Transfer Protocols section] for more details. 
-  Otherwise, use the file system path necessary. Ex D:\SatisfactoryDS or /opt/SatisfactoryDedicatedServer
-  NOTE: Entering servers by network filesystem path is not currently an option in SMM, use ficsit-cli in the mean time.
+4. Enter server connection details in the popup window.
+  a. link:#FileTransferMethods[File Transfer Methods] section has more information regarding FTP/SFTP configurations.
```
Comment (mircearoata) [inline modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc:70]:
```suggestion
  a. The link:#FileTransferMethods[File Transfer Methods] section has more information regarding FTP/SFTP configurations.
```

**Review by mircearoata — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc [hunk 10]
```diff
@@ -87,8 +95,62 @@ Note that applying changes in ficsit-cli is a global action -
...
+====
+
+Server Message Block (SMB) is also commonly known as Windows File Shares or CIFS (Common Internet File System) is a common method to transfer files over a Network, commonly on Windows Systems, but may be used on Linux/Unix Systems. 
+The commonly used port number is TCP/445.
+The examples provided are for a self-hosted dedicated server.
+
+The authenticating user requires Read/Write/Delete or Read/Modify permissions.
+
+The path required for SMB/CIFS to be used is the following:
+`\\ServerNameOrIP\ShareName\Path`
```
Comment (mircearoata) [inline modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc:147]: For Linux it should be `//ServerNameOrIP/ShareName/Path`, right? So both should be mentioned

**Pull request merged**